awx/ui/client/test/e2e/tests/test-auditor-read-only-forms.js

```diff
@@ -0,0 +1,444 @@
+                adminOrganization,
+                adminInventoryScript,
+                adminNotificationTemplate,
+                adminProject,
+                adminSmartInventory,
+                adminStandardInventory,
+                adminTeam,
+                adminUser,
+                auditor
+           };
+
+           client.login(store.auditor.username, store.auditor.password);
+
+            done();
+        });
+    },
+    'verify an auditor\'s credentials inputs are read-only': function (client) {
+        navigateAndWaitForSpinner(client, `${credentials.url()}/${store.created.adminAWSCredential.id}/`);
+
+        credentials.section.edit
+            .expect.element('@title').text.contain(store.created.adminAWSCredential.name);
+
+        checkDisabledElements(client, ['.at-Input']);
+    },
+    'verify an auditor\'s inventory scripts inputs are read-only': function (client) {
+        navigateAndWaitForSpinner(client, `${inventoryScripts.url()}/${store.created.adminInventoryScript.id}/`);
+
+        inventoryScripts.section.edit
+            .expect.element('@title').text.contain(store.created.adminInventoryScript.name);
+
+        let selectors = [
+            '#inventory_script_form .Form-textInput',
+            '#inventory_script_form .Form-textArea',
+            '#inventory_script_form .Form-lookupButton'
+        ];
+
+        checkDisabledElements(client, selectors);
+    },
+    'verify save button hidden from auditor on inventory scripts form': function () {
+        inventoryScripts.expect.element('@save').to.not.be.visible;
+    },
+    'verify an auditor\'s job template inputs are read-only': function (client) {
+        navigateAndWaitForSpinner(client, `${templates.url()}/job_template/${store.created.adminJobTemplate.id}/`);
+
+        templates.section.editJobTemplate
+            .expect.element('@title').text.contain(store.created.adminJobTemplate.name);
+
+        client.pause(2000);
+
+        let selectors = [
+            '#job_template_form .Form-textInput',
+            '#job_template_form select.Form-dropDown',
+            '#job_template_form .Form-textArea',
+            '#job_template_form input[type="checkbox"]',
+            '#job_template_form .ui-spinner-input',
+            '#job_template_form .ScheduleToggle-switch'
+        ];
+
+        checkDisabledElements(client, selectors);
+    },
+    'verify save button hidden from auditor on job templates form': function () {
+        templates.expect.element('@save').to.not.be.visible;
+    },
+    'verify an auditor\'s notification templates inputs are read-only': function (client) {
+        navigateAndWaitForSpinner(client, `${notificationTemplates.url()}/${store.created.adminNotificationTemplate.id}/`);
+
+        notificationTemplates.section.edit
+            .expect.element('@title').text.contain(store.created.adminNotificationTemplate.name);
+
+        let selectors = [
+            '#notification_template_form .Form-textInput',
```

Comment:
We should pull this logic into a generic "checkAllFieldsDisabled" command for form sections. You could add this custom command here: https://github.com/ansible/awx/blob/devel/awx/ui/client/test/e2e/objects/sections/createFormSection.js#L89

something like this (probably won't run as is, but I think you'll get the idea)
```
const checkAllFieldsDisabled = function() {
    let possibleInputs = [
        '.at-Input',
        '.Form-textInput',
        'select.Form-dropDown',
        'input[type="checkbox"]',
        'input[type="radio"]',
        '.ui-spinner-input',
        '.Form-textArea',
        '.ScheduleToggle-switch',
        '.Form-lookupButton'
    ];
    possibleInputs.forEach(function(selector) {
        // ...
    })
};

const createFormSection = function({ selector, labels, strategy }) {
    let options = generatorOptions[strategy || 'default'];

    let formSection = {
        selector,
        sections: {},
        elements: {},
        commands: [{
            checkAllFieldsDisabled
        ]}
    };
```

This way, we could reuse most of the selectors already created on each of the form sections. In order to do this, we'll probably need some sort of "